tracee/printer.go

```diff
@@ -47,6 +50,15 @@ func newEventPrinter(kind string, containerMode bool, out io.Writer, err io.Writ
 			err: gob.NewEncoder(err),
 		}
 	}
+
+	if strings.HasPrefix(kind, "go-template=") {
```

Comment:
is there a particular reason this is outside the `switch` block? if not, please make it a switch case

Reply:
@itaysk yeah because I'm don't think the switch statement supports pattern matching but instead only a full equality check. 
There's also the boolean version of switch for regex matching but all the other cases will have to be rewritten as a boolean condition for equality instead :(

Reply:
doesn't there's a problem with including the prefix check in the switch: https://play.golang.org/p/MWWHC5oWV4A

Reply:
Okay I wasn't sure if you'd be okay with a boolean switch with `case kind == value` .... I'll make the change then.

Reply:
oh I understand what you meant now by boolean switch. Yes I think it's better this way because we are expecting to add more output formats which have arguments in the future